### PR TITLE
Dec 21 '23 last changes before audit.

### PR DESCRIPTION
**Pull request by codinghistorian**
A list of commits.

FXD-72 '23 Dec 21 clean indexed event param changes
FXD-72 '23 Dec 21 clean comments on test script
FXD-72 '23 Dec 21 remove deprecated storage slots
FXD-72 '23 Dec 21 adding event to WL, setBookKeeper fns
FXD-72 '23 Dec 21 adding event to setAccessControlConfig fns
FXD-72 '23 Dec 21 adding comment to setDecentralizedStatesStatus fn

### CODE REVIEW

File: contracts/main/stablecoin-core/LiquidationEngine.sol [hunk 1]
```diff
@@ -38,14 +38,15 @@ contract LiquidationEngine is PausableUpgradeable, ReentrancyGuardUpgradeable, I
...
 
-    bytes32 internal deprecated;
-
     IBookKeeper public bookKeeper; // CDP Engine
     ISystemDebtEngine public systemDebtEngine; // Debt Engine
     uint256 public override live; // Active Flag
     mapping(address => bool) public liquidatorsWhitelist;
 
     event LiquidationFail(bytes32 _collateralPoolIds, address _positionAddresses, address _liquidator, string reason);
+    event LogSetBookKeeper(address indexed _newAddress);
```
Comment (BaldyAsh) [inline contracts/main/stablecoin-core/LiquidationEngine.sol:47]: why to keep it indexed? you kept access control address non-indexed in its event

File: contracts/main/stablecoin-core/PriceOracle.sol [hunk 2]
```diff
@@ -39,6 +39,7 @@ contract PriceOracle is CommonMath, PausableUpgradeable, IPriceOracle, ICagable,
     );
 
     event LogSetStableCoinReferencePrice(address indexed _caller, uint256 _data);
+    event LogSetBookKeeper(address indexed _newAddress);
```
Comment (BaldyAsh) [inline contracts/main/stablecoin-core/PriceOracle.sol:42]: .

File: contracts/main/stablecoin-core/liquidation-strategies/FixedSpreadLiquidationStrategy.sol [hunk 3]
```diff
@@ -72,6 +72,7 @@ contract FixedSpreadLiquidationStrategy is CommonMath, PausableUpgradeable, Reen
         uint256 _treasuryFees
     );
     event LogSetFlashLendingEnabled(address indexed _caller, uint256 _flashLendingEnabled);
+    event LogSetBookKeeper(address indexed _newAddress);
```
Comment (BaldyAsh) [inline contracts/main/stablecoin-core/liquidation-strategies/FixedSpreadLiquidationStrategy.sol:75]: .

**Review by BaldyAsh — CHANGES_REQUESTED** [review]
Please run "npm run linter && npm run prettier" after you finish

File: contracts/main/stablecoin-core/LiquidationEngine.sol (hunk 1 above)
Comment (codinghistorian) [reply contracts/main/stablecoin-core/LiquidationEngine.sol:47]: Actually, no need. Valid point. newAddress for bookKeper is not in critical need for later search. Will make it un indexed.

File: contracts/main/stablecoin-core/PriceOracle.sol (hunk 2 above)
Comment (codinghistorian) [reply contracts/main/stablecoin-core/PriceOracle.sol:42]: will make it un indexed.

File: contracts/main/stablecoin-core/liquidation-strategies/FixedSpreadLiquidationStrategy.sol (hunk 3 above)
Comment (codinghistorian) [reply contracts/main/stablecoin-core/liquidation-strategies/FixedSpreadLiquidationStrategy.sol:75]: will make it un indexed.

**Pull request merged**